app/main.ipynb

```diff
@@ -336,43 +440,1073 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 18,
+   "source": [
+    "# Python code\n",
+    "%time [i for i in range(1000000)]"
```

Comment:
The use of list comprehension to create a list of numbers from 0 to 999999 is correct for the Python implementation.